projects/gyroscope/index.js

```diff
@@ -1,25 +1,24 @@
-
-const { getLogs, getAddress } = require('../helper/cache/getLogs')
 const { transformBalances } = require('../helper/portedTokens')
 const sdk = require('@defillama/sdk')
 
+const GYRO_POOL_ADDRESSES = [
```

Comment:
would be nice if we could get this list on chain or via some api so it doesnt become stale

Reply:
nice will do that!